projects/sbb-esta/angular-business/dialog/src/dialog/dialog-footer/dialog-footer.component.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  HostBinding,
+  Input,
+  OnInit,
+  Optional,
+  ViewEncapsulation
+} from '@angular/core';
+
+import { DialogHelperService } from '../dialog-helper.service';
+import { DialogRef } from '../dialog-ref';
+import { Dialog } from '../dialog.service';
+
+/**
+ * Container for the bottom action buttons in a dialog.
+ * Stays fixed to the bottom when scrolling.
+ */
+@Component({
+  selector: `sbb-dialog-footer, [sbbDialogFooter]`,
+  styleUrls: ['./dialog-footer.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  template: `
+    <perfect-scrollbar class="sbb-dialog-footer-scrollbar">
```

Comment:
After some careful thought, I question whether we should support scrolling in the footer.
The footer is only intended to have buttons placed inside them, which should not exceed the available space.
The consequence would be removing the perfect-scrollbar in the template here.
What do you think?

Reply:
I agree! I removed the scrollbar.